src/crypto/index.js

```diff
@@ -181,6 +189,115 @@ Crypto.prototype.init = async function() {
         );
         this._deviceList.saveIfDirty();
     }
+
+    this._checkAndStartKeyBackup();
+};
+
+/**
+ * Check the server for an active key backup and
+ * if one is present and has a valid signature from
+ * one of the user's verified devices, start backing up
+ * to it.
+ */
+Crypto.prototype._checkAndStartKeyBackup = async function() {
+    console.log("Checking key backup status...");
+    let backupInfo;
+    try {
+        backupInfo = await this._baseApis.getKeyBackupVersion();
+    } catch (e) {
+        console.log("Error checking for active key backup", e);
+        if (Number.isFinite(e.httpStatus) && e.httpStatus / 100 === 4) {
```

Comment:
when would you ever have something which is divisible by 100 to give 4... but isn't a finite number?

Reply:
I think this was essentially checking it's a number as opposed to null / undefined, although either can be divided by 100 to give something that is not 4, so arguably unnecessary.